worker/include/RTC/RTCP/CompoundPacket.hpp

```diff
@@ -68,6 +71,11 @@ namespace RTC
 			this->sdesPacket.AddChunk(chunk);
 		}
 
+		inline void CompoundPacket::AddReceiverExtendedReport(ReceiverExtendedReport* report)
+		{
+			this->receiverExtendedReportPacket.AddReport(report);
```

Comment:
This packet does not have the SSRC set has in the [spec](https://tools.ietf.org/html/rfc3611#section-2). Why is that?